.github/workflows/docs-ci.yaml

```
@@ -0,0 +1,36 @@
+name: Docs CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

I think that `on>push>main` like this means it's only tracking pushes to `main`. If we replace the `branches` part with `paths` it _should_ check all pushes to all branches as long as they touch files in `docs`

```suggestion
on:
    push:
        paths:
            - "docs/**"
    pull_request:
        paths:
            - "docs/**"
```

---

We are also linting files outside of `docs` for example README.md, CONTRIBUTING.md SECURITY.md etc. So I think the current scope is fine. Or we have to expand the `paths` scope and include all those files too which could cause a drift if we add more markdown files outside of `docs/` in the future.

---

Addressed.